_pages/infrastructure/sandbox.md

```diff
@@ -34,27 +34,27 @@ sure you tag all resources you create in the AWS sandbox with the tag
 the name you registered with
 [#infrastructure](https://gsa-tts.slack.com/messages/C039MHHF8/).
 
-* [Request a sandbox account](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSeXlsfDyAZi5P_5lS8InCeZxvA3JA3-g8aYWdkgkFmGPcYmLQ/viewform)
-* [Sign in to your sandbox account](https://tts-sandbox.signin.aws.amazon.com/console)
+- [Request a sandbox account](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSeXlsfDyAZi5P_5lS8InCeZxvA3JA3-g8aYWdkgkFmGPcYmLQ/viewform)
+- [Sign in to your sandbox account](https://tts-sandbox.signin.aws.amazon.com/console)
 
 ## Important notes for AWS users
 
 There are a few special notes on using any "Infrastructure as a Service" in the Federal context.
 
 ### Other people's money
 
-The federal government cannot pay one penny more than it is authorized to spend. There is no retroactive justification for spends. When government exceeds these limits, a report and explanation is required to the GSA Administrator, General Counsel, and Congress. So tracking costs is a *big deal*.
+The federal government cannot pay one penny more than it is authorized to spend. There is no retroactive justification for spends. When government exceeds these limits, a report and explanation is required to the GSA Administrator, General Counsel, and Congress. So tracking costs is a _big deal_.
 
-However we recognize that it's important to provide compute resources for 18F
+However we recognize that it's important to provide compute resources for TTS
```

Comment:
Should that be _computing_ resources?

Reply:
Either is correct. ["Compute"](https://www.techopedia.com/definition/6580/compute) is the term I hear more in the devops world, to differentiate from storage, networking, etc.